File: benchmark/results/java-multilang/run-004/b-baseline/output/com/metaengine/demo/identity/services/TokenService.java
package com.metaengine.demo.identity.services;

import java.util.List;

import com.metaengine.demo.identity.aggregates.User;

/** TokenService domain service. */
public class TokenService {

    /** Issue a token for a User. */
    public User create(User input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Find a User by id (may return null). */
    public User findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List users up to limit. */
    public List<User> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Delete a User by id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}

File: benchmark/results/python-invocation-script/run-005/a-mcp/output/identity/services/token_service.py
from identity.aggregates.user import User

"""TokenService service."""
class TokenService:
    def create(self, input: User) -> User:
        raise NotImplementedError('not implemented')
    def findById(self, id: str) -> User | None:
        raise NotImplementedError('not implemented')
    def list(self, limit: float) -> list[User]:
        raise NotImplementedError('not implemented')
    def delete(self, id: str) -> None:
        raise NotImplementedError('not implemented')

